src/docfx/lib/schema/JsonSchemaTransformer.cs

```diff
@@ -424,6 +429,15 @@ private string GetXrefHref(FilePath file, string uid, int uidCount, bool isRootL
 
                         errors.AddIfNotNull(xrefError);
 
+                        if (xrefSpec != null &&
+                        schema.XrefType != null &&
```

Comment:
Use the same indention as line 432, keep them left align (without `if (`)

Reply:
OK